code/modules/admin/outfit_editor.dm

```diff
@@ -178,7 +178,7 @@
 		if("l_hand")
 			choose_any_item(slot)
 		if("back")
-			options = typesof(/obj/item/storage/backpack)
+			options = typesof(/obj/item/storage/backpack) + typesof(/obj/item/mod/control/pre_equipped)
```

Comment:
what if the modsuit doesnt fit on the back

Reply:
Added a check to make sure only back-fitting MODs work